docs/index.rst

```diff
@@ -48,6 +48,31 @@ The result is returned as an OrderedDict, containing PacketField names as keys a
    result = pkt.load('MyCCSDS.bin')
 
 
+Splitting Mixed Streams by APID
+-------------------------------
+Often, CCSDS data with arrive from external sources into software systems in a single file with multiple APID's. Splitting a mixed file or stream of bytes by APID so they can be used with the `ccsdspy.FixedLength` class can be done thorugh the API or with the module command line interface.
+
+.. code-block:: python
+
+  import ccsdspy
+
```

Comment:
Add: `from ccsdspy.util import split_by_apid`, otherwise the code example doesn't run.